src/main/lpfConverter.ts

```diff
@@ -201,24 +368,95 @@ function w3cPublicationManifestToReadiumPublicationManifest(w3cManifest: JsonMap
     }
     {
-        const readingOrder = pop("readingOrder") as JsonMap;
+        const readingOrder = pop("readingOrder");
         const links = convertW3CpublicationLinksToReadiumManifestLink(readingOrder);
         if (links.length) {
             publication.Spine = links;
             // https://github.com/readium/r2-shared-js/blob/develop/src/models/publication.ts#L63
         }
     }
     {
-
+        const raw = pop("readBy");
+        const contrib = convertW3cEntitiesToReadiumManifestContributors(raw);
+        if (contrib.length) {
+            publication.Metadata.Narrator = contrib;
+        }
+    }
+    {
+        const raw = pop("author");
+        const contrib = convertW3cEntitiesToReadiumManifestContributors(raw);
+        if (contrib.length) {
+            publication.Metadata.Author = contrib;
+        }
+    }
+    {
+        const raw = pop("publisher");
+        const contrib = convertW3cEntitiesToReadiumManifestContributors(raw);
+        if (contrib) {
+            publication.Metadata.Publisher = contrib;
+        }
+    }
+    {
+        const raw = pop("accessMode");
+        const rawArray = (Array.isArray(raw) ? raw : [raw]);
+        const rawArrayFiltered = rawArray.filter((l) => typeof l === "string");
+        if (rawArrayFiltered.length) {
+            publication.Metadata.AccessMode = rawArrayFiltered;
+        }
+    }
+    {
+        const raw = pop("accessModeSufficient");
+        const rawArray = (Array.isArray(raw) ? raw : [raw]);
+        const rawArrayFormated = rawArray
+            .reduce(
+                (pv, cv) => [
+                    ...pv,
+                    Array.isArray(cv?.itemListElement)
+                        ? cv.itemListElement.join(",")
+                        : undefined,
+                ], [])
+            .filter((f: any) => f);
+
+        if (rawArrayFormated.length) {
+            publication.Metadata.AccessModeSufficient = rawArrayFormated;
+        }
+    }
+    {
+        const raw = pop("accessibilityFeature");
+        const rawArray = (Array.isArray(raw) ? raw : [raw]);
+        const rawArrayFiltered = rawArray.filter((l) => typeof l === "string");
+        if (rawArrayFiltered.length) {
+            publication.Metadata.AccessibilityFeature = rawArrayFiltered;
+        }
+    }
+    {
+        const raw = pop("accessibilityHazard");
+        const rawArray = (Array.isArray(raw) ? raw : [raw]);
+        const rawArrayFiltered = rawArray.filter((l) => typeof l === "string");
+        if (rawArrayFiltered.length) {
+            publication.Metadata.AccessibilityHazard = rawArrayFiltered;
+        }
+    }
+    {
+        const raw = pop("accessibilitySummary");
+        const loc = convertW3cLocalisableStringToReadiumManifestTitle(raw);
+        if (loc) {
+            const locMap = (Array.isArray(loc) ? loc : { und: loc }) as IStringMap;
+            const summary = Object.values(locMap);
+            publication.Metadata.AccessibilitySummary = summary;
```

Comment:
In `r2-shared-js`, the `schema:accessibilitySummary` EPUB2/3 metadata is now correctly parsed as a potentially-multilingual string (i.e. union type of plain `string`, or `IStringMap` with language keys and default to `und`).
My next commit to the develop branch will include the updated package.

Reply:
The develop branch is now updated. `publication.Metadata.AccessibilitySummary` is now `string | IStringMap`, please update the conversion code accordingly (if there is only a single `accessibilitySummary` in the W3C document without specific language / default language, then `string` should be favoured instead of the language map).

https://github.com/readium/readium-desktop/commit/9e3b13a84e5ef0f3bdbb506e73c50e59719b1cb4

https://github.com/readium/r2-shared-js/compare/v1.0.34...v1.0.35

Reply:
https://github.com/readium/readium-desktop/pull/1009/commits/31453cf6a1f6afcd723f92c27ee6dcdcfc0439bb